src/makeCalculator.js

```diff
@@ -38,6 +38,35 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(a) {
+      this.result += a;
+    },
```

Comment:
```suggestion
    add(a) {
      this.result += a;
      
      return this;
    },
```

Reply:
The task says that only the 'operate' and 'reset' methods are called by the chain. return.this don't use in another methods